.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -6,50 +6,49 @@ week: 1
 step: 4
 name: Week 1 Step 4
 ---
+Week 1 Step 4 ⬤⬤⬤⬤◯◯◯◯ | 🕐 Estimated completion: 35mins-45 mins
+### Writing Your First Serverless Function
+It's Time! to write your first Serverless function. 
 
-## Week 1 Step 4
 
-🕐 Estimated completion: 35-45mins
+### ✅  Task:
+- [ ]  run `git pull` 
+- [ ]  make sure you are in the `week1` branch 
+- [ ]  place your function URL in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `HTTP_ENDPOINT`
+- [ ]  create and deploy a HTTP trigger Azure Function that outputs the current time and date.
+- [ ] Copy and paste your function code to a file `week1/httptime.js` 
+- [ ] Commit your changes to move on 
 
-### Writing Your First Function
-
-## Configuring Azure
-
-*This week, you will be going through steps to set up tools needed to be successful in this camp. If you are already familiar with some, feel free to skip to the end and complete the task to move on.*
-
-### The Cloud
+*Note: Every time you make a new commit to `week1`, we will check your function by making a request to it*
 
+### 🚧 Test your Work
+Use **Postman**! Paste the *function url* and make a GET request. In the output box, you should get something like this:
+```
+Hello! The current time is: 17 : 41 : 34 PM
+```
 <details>
-<summary>:question: So what is "The Cloud"?</summary>
+    <summary> How to use Postman to test. </summary>
     </br>
-    
-![http://infomotions.com/musings/waves/media/client-server-illustration.gif](http://infomotions.com/musings/waves/media/client-server-illustration.gif)
+    <img src="https://user-images.githubusercontent.com/69332964/113361613-63ef6c00-931a-11eb-9380-7d58a3dea1b4.png" width=600px/>
+</details>
 
-> A [server](https://www.infotech.co.uk/blog/it-infrastructure-what-does-a-server-actually-do) is a computer that provides (serves) data to other computers, called clients. Client connect to servers through the internet.
+### What is Serverless?  
+Similarly, serverless functions are cool since they engage the use of serverless computing without the customer (you) having to worry about server space. This allows you to deploy code quickly and easily.
```

Comment:
This should be in its own issue and be the first thing in week 1 since students should understand what serverless is before starting to code.